subsys/bluetooth/host/conn.c

```diff
@@ -1178,6 +1178,23 @@ int bt_conn_send_cb(struct bt_conn *conn, struct net_buf *buf,
 	return 0;
 }
 
+static bool conn_tx_internal(bt_conn_tx_cb_t cb)
+{
+#if defined(CONFIG_BT_L2CAP_DYNAMIC_CHANNEL)
```

Comment:
Can we have a single `if()` statement with a conditional `#if defined()` that only disables the `l2cap_chan_sdu_sent`? this is needless duplication

Reply:
Sure will change it.

Reply:
Done.

Reply:
`att_req_sent` still duplicated maybe change the order so that `cb == l2cap_chan_sdu_snet ||` can be on a single line?

Reply:
Not sure I follow what is the problem of duplicating the condition here, even if I reorganize the checks it won't fit in a single line and that would help either since the entire line would be enclosed into a #if Id have to duplicate it again on the #else so att_req_sent being on the next line actually helps reducing the duplicated code or you want some construct like: att_req_sent #if ... || cb == l2cap_chan_sdu_sent #endif ) { (inside the line #if is probably not recommended either)

Reply:
I meant like this:
```
	if (cb == att_pdu_sent || cb == att_cfm_sent || cb == att_rsp_sent ||
#if defined(CONFIG_BT_L2CAP_DYNAMIC_CHANNEL)
	    cb == l2cap_chan_sdu_sent ||
#endif
	    cb == att_req_sent) {
```
It's no big issue here, but I've seen worse examples.

I prefer this:
https://github.com/zephyrproject-rtos/zephyr/blob/master/subsys/bluetooth/controller/ll_sw/ctrl.c#L9696
in favour of this:
https://github.com/zephyrproject-rtos/zephyr/blob/master/subsys/bluetooth/controller/ll_sw/ctrl.c#L2909

Reply:
This is breaking codying style then since you are adding a line break when there would be space for adding another condition.

Reply:
I don't see that rule anywhere. However it's not important, just my two cents.

Reply:
It doesn't seem checkpath complains so I did the change.